README.md

```diff
@@ -1,38 +1,325 @@
-have to do that.
+# Installation
+
+## Aquedux
+
+If you use Yarn
+```js
+// Client side (aka browser & Redux app)
+yarn add aquedux-client
+// or
+npm i aquedux-client
+// Server side (aka Node.js & Redux app)
+yarn add aquedux-server
+// or
+npm i aquedux-server
+```
 
-With Aquedux, you simply share the reducer code across the client and
-the server app, describe how Redux actions are listening to and "Voila!".
+## Redis
 
-It makes writing client(s)/server applications easy without adding another layer by leveraging the elegant design of Redux.
+For the aquedux-server app to work, you have to install a Redis server on your machine. Don't worry, if you have never installed one, it is extremly simple:
 
-# Installation
+* On OSX: Install it with brew with `brew install redis-server`
+* On Linux: Install it with your system package manager. For instance, if it is apt, with `apt-get install redis-server`
+* On Windows: For windows you have two options, either download it from the redis website, or install it on your `Ubuntu On Windows`.
 
-```sh
-// node server side
-yarn add aquedux-server
-// or browser side
-yarn add aquedux-client
+You don't have to configure anything, just running it.
+You now should have a running instance on the localhost:XXXX port.
+
+# Getting Started
+
+Let see the required steps to integrate Aquedux in your current Redux workflow for the client and the server app.
+
+## The client app
+
+```js
+/****************************
+*    configureAquedux.js    *
+****************************/
+
+import { createAqueduxClient } from 'aquedux-client'
+
+const configureAquedux = (store, endpoint) => {
+  // Set actionTypes that should be sent over Aquedux and its enpoint
+  const aqueduxOptions = {
+    hydratedActionTypes: ['ADD_TODO', 'TOGGLE_TODO'],
+    endpoint
+  }
+
+  // Create the client instance.
+  const client = createAqueduxClient(store, aqueduxOptions)
+
+  // Declare a channel with a way to reduce its snapshot. This is used to group action types and store slices.
+  // The reducer tells Aquedux how to reduce into your state the slice sent over by Aquedux as the channel
+  // initial state.
+  client.addChannel('todos', (oldState, action) => {
+    return {
+      ...oldState,
+      todos: action.snapshot
+    }
+  })
+
+  return client
+}
+
+export default configureAquedux
+
+/****************************
+*    configureStore.js      *
+****************************/
+
+import { createStore, subscribeToChannel } from 'aquedux-client'
+import configureAquedux from './configureAquedux'
+import rootReducer from './reducers'
+
+const configureStore = () => {
+  const store = createStore(rootReducer)
+
+  // The default route served by aquedux-server is $YOUR_HOST/aquedux
+  const endpoint = `${protocol}://${host}:${port}/aquedux`
```

Comment:
protocol, host, port are unknown variables.
I think this example should be simpler, no need to separate logic in 2 files imo.